docs/modules/reference/pages/daemons/daemon-config-files/provisiond.adoc

```diff
@@ -75,17 +75,23 @@ NOTE: Any changes to the thread pools require a restart of the {page-component-t
 This pool determines the number of requisitions that can be imported at one time.
 Increase this number if you have a large number of requisitions.
 
+Import threads will check the requisition against the database to determine what add/change/delete operations are necessary.
+They will also attempt to pull SNMP system information from the node into the database.
+Once this inventory update is complete, the import thread will schedule the node for scanning/rescanning as necessary.
```

Comment:
... will schedule the Node scan.

Reply:
"as necessary" based on if the sync is set as all/new/dbOnly
```suggestion
Once this inventory update is complete, the import thread will schedule a node scan as necessary.
```